packages/button/docs/Button.stories.tsx

```diff
@@ -158,3 +158,17 @@ export const Working: StoryFn = () => (
  * `import { LoadingInput } from "@kaizen/loading-skeleton"`
  */
 export const Loading: StoryFn = () => <LoadingInput isAnimated width={13} />
+
+/**
+ * <p>Button now extends native HTML "form" attributes for `button`</p>
+ * <p>You can use this to submit a `form` using `Button` with a matching form id</p>
+ */
+export const NativeFormButton: StoryFn = () => (
```

Comment:
This story probably needs a `<form>` in it to showcase the connection :)

Reply:
Fair point, probably a nice way to do a quick demo for our consumers 😉